modules/standard/IO.chpl

```diff
@@ -1255,16 +1255,22 @@ record file {
   var home: locale = here;
   pragma "no doc"
   var _file_internal:qio_file_ptr_t = QIO_FILE_PTR_NULL;
+
+  // INIT TODO: This would be a useful case for requesting a default initializer
+  // be built even when handwritten initializers (copy init) exist.
+  proc init() {
+  }
 }
 
 // TODO -- shouldn't have to write this this way!
 pragma "no doc"
-pragma "init copy fn"
-proc chpl__initCopy(x: file) {
-  on x.home {
-    qio_file_retain(x._file_internal);
+proc file.init(x: file) {
+  this.home = x.home;
+  this._file_internal = x._file_internal;
+  this.complete();
+  on home {
+    qio_file_retain(_file_internal);
```

Comment:
Just curious - why is the 'this.complete' needed, if it is?

Reply:
Hmm, not sure... I thought it was needed, but trying out a few tests it doesn't appear to be necessary.